releasenotes/notes/note-for-imperfect-analysis-dc49f2c07de327f4.yaml

```
@@ -0,0 +1,10 @@
+---
+issues:
+  - |
+    Cureve analysis may have imperfection in the uncertainty propagation computation.
+    Fit paramters consist of the nominal part and standard error, however,
+    the correlation of these paramters are not precisely taken into account.
+    This sometimes result in the overestimation of the confidence interval of fit curves,
+    or overestimation of the standard error of some analysis values computed with
+    multiple fitting parameters. This issue will be solved in the version 0.3.
+    See qiskit-experiments/#551 for details.
```

You can make this a hyperlink to the PR
```suggestion
    See [qiskit-experiments/#551](https://github.com/Qiskit/qiskit-experiments/pull/551) for details.
```